tests-config/setup.js

```diff
@@ -1,52 +1,71 @@
-'use strict'
+"use strict";
 
-const fs = require('fs')
-const { extname } = require('path')
-const unified = require('unified')
-const parse = require('remark-parse')
-const stringify = require('remark-stringify')
-const frontmatter = require('remark-frontmatter')
-const footnotes = require('remark-footnotes')
-const pangu = require('../index')
+const fs = require("fs");
+const { extname } = require("path");
+const unified = require("unified");
+const parse = require("remark-parse");
+const stringify = require("remark-stringify");
+const frontmatter = require("remark-frontmatter");
+const footnotes = require("remark-footnotes");
+const pangu = require("../index");
 
-const processor = unified()
-  .use(parse, { footnotes: true })
-  .use(stringify)
-  .use(frontmatter)
-  .use(footnotes, {inlineNotes: true})
-  .use(pangu)
-  .freeze()
+const processor = function (options) {
+  return unified()
+    .use(parse, { footnotes: true })
+    .use(stringify)
+    .use(frontmatter)
+    .use(footnotes, { inlineNotes: true })
+    .use(pangu, options)
+    .freeze();
+};
 
-function formatAsync(source) {
-  return processor.process(source)
-}
+const allOptionsOffProcessor = processor({
+  text: true,
+  inlineCode: true,
+  link: true,
+  image: true,
+  imageReference: true,
+  definition: true,
+});
 
 function raw(string) {
-  if (typeof string !== 'string') {
-    throw new Error('Raw snapshots have to be strings.')
+  if (typeof string !== "string") {
+    throw new Error("Raw snapshots have to be strings.");
   }
-  return { [Symbol.for('raw')]: string }
+  return { [Symbol.for("raw")]: string };
 }
 
-function runSpec(dirname) {
-  fs.readdirSync(dirname).forEach(filename => {
-    const path = dirname + '/' + filename
+function runSpec(dirname, options) {
+  fs.readdirSync(dirname).forEach((filename) => {
+    const path = dirname + "/" + filename;
     if (
-      extname(filename) !== '.snap' &&
+      extname(filename) !== ".snap" &&
       fs.lstatSync(path).isFile() &&
-      filename !== 'run.spec.js'
+      filename !== "run.spec.js"
     ) {
-      const source = fs.readFileSync(path)
+      const source = fs.readFileSync(path);
       test(`${filename}`, () => {
```

Comment:
测试的名称里可以把 options 加上，看 snapshot 的时候更易读一些：

```suggestion
      test(`${filename} with options: ${JSON.stringify(options)}`, () => {
```